SecretManagement.KeePass.Extension/Tests/Get-Secret.Tests.ps1

```diff
@@ -0,0 +1,185 @@
+Import-Module -Name 'Microsoft.PowerShell.SecretManagement'
+Import-Module -Name "$($PSScriptRoot)/../../SecretManagement.KeePass.psd1" -Force
+
+InModuleScope -ModuleName 'SecretManagement.KeePass.Extension' {
```

Comment:
Related to the beforeAll, the Inmodulescope should be after the describe definition, not before.

Reply:
Same here